src/Route.elm

```diff
@@ -19,34 +21,35 @@ type Route
     | Options (Maybe String) (Maybe String) (Maybe String) (Maybe Int) (Maybe Int) (Maybe String)
 
 
+{-| Fixes issue with elm/url not properly escaping string
+-}
+queryString : String -> Query.Parser (Maybe String)
+queryString =
+    Query.map (Maybe.andThen Url.percentDecode) << Query.string
+
+
```

Comment:
Since [Domen's PRs](https://github.com/elm/url/pull/34) are still not merged we need to define fixed version ourselves.

Reply:
:100: